basis/formatting/formatting.factor

```diff
@@ -113,9 +113,9 @@ fmt-C     = "C"                  => [[ [ 1string >upper ] ]]
 fmt-s     = "s"                  => [[ [ present ] ]]
 fmt-S     = "S"                  => [[ [ present >upper ] ]]
 fmt-u     = "u"                  => [[ [ unparse ] ]]
-fmt-d     = "d"                  => [[ [ >integer number>string ] ]]
-fmt-o     = "o"                  => [[ [ >integer >oct ] ]]
-fmt-b     = "b"                  => [[ [ >integer >bin ] ]]
+fmt-d     = "d"                  => [[ [ number>string ] ]]
+fmt-o     = "o"                  => [[ [ >oct ] ]]
+fmt-b     = "b"                  => [[ [ >bin ] ]]
 fmt-e     = digits "e"           => [[ first '[ _ format-scientific ] ]]
```

Comment:
In Python, ``%d`` and ``%x`` and ``%o`` format integers:

```
>>> '%d' % 123.456
'123'
>>> '%x' % 123.456
'7b'
>>> '%o' % 123.456
'173'
```

I realize, they don't have our ``%x`` for floats, so maybe ``%a`` is a better approach?